pylib/gyp/common.py

```diff
@@ -422,17 +422,62 @@ def EnsureDirExists(path):
     except OSError:
         pass
 
+def GetCrossCompilerPredefines():
```

Comment:
Let's always return a list.  Functions that return `list | None` put more work on the caller.
```suggestion
def GetCrossCompilerPredefines():
```